views/old_settings.html

```diff
@@ -15,7 +15,7 @@ <h1 class="title"><%= __('Settings') %></h1>
     >
       <!-- tabs -->
       <h2 class="nav-tab-wrapper" id="mailpoet_settings_tabs">
-        <a class="nav-tab" href="#basics" data-automation-id="basic_settings_tab"><%= __('Basics') %></a>
+        <a class="nav-tab" href="?page=mailpoet-new-settings#/basics" data-automation-id="basic_settings_tab"><%= __('Basics') %></a>
```

Comment:
Can we set the URL for the new version to `?page=mailpoet-settings#/basics` and use `?page=mailpoet-legacy-settings#/basics` ?